### ISSUE TITLE
Bottom tabs text is cut off on iPad

### ISSUE THREAD

**Issue by d4vidi** [wix/react-native-navigation#7855]
### What happened?

Complete info: `WOARNN-568`

The text in some of the bottom tabs is cut off -

![image](https://github.com/wix/react-native-navigation/assets/9818880/90abe344-3859-490c-a941-752a20e12acb)

### Was it tested on latest react-native-navigation?

- [X] I have tested this issue on the latest react-native-navigation release and it still reproduces.

### In what environment did this happen?

React Native Navigation version: 
React Native version: 
Has Fabric (React Native's new rendering system) enabled: (yes/no) 
Node version: 
Device model: 
iOS version:

**Comment by phuongwd** [comment]
Related to

1. https://github.com/wix/react-native-navigation/issues/7506
2. https://github.com/wix/react-native-navigation/issues/7827

Patch

```
diff --git a/node_modules/react-native-navigation/lib/ios/RNNTabBarItemCreator.m b/node_modules/react-native-navigation/lib/ios/RNNTabBarItemCreator.m
index d313bfc..c22f984 100644
--- a/node_modules/react-native-navigation/lib/ios/RNNTabBarItemCreator.m
+++ b/node_modules/react-native-navigation/lib/ios/RNNTabBarItemCreator.m
@@ -91,13 +91,19 @@
     NSNumber *fontSize = [bottomTabOptions.fontSize withDefault:@(10)];
     NSString *fontWeight = [bottomTabOptions.fontWeight withDefault:nil];
 
+    /**
+     * The title of tabbar item has been cut off in iPad 12.9 with iOS 15.4.
+     * It was mentioned at https://github.com/wix/react-native-navigation/issues/7506
+     * Hotfix: centered the text will resolve problem.
+     */
+
     NSDictionary *normalAttributes = [RNNFontAttributesCreator
         createFromDictionary:[tabItem titleTextAttributesForState:UIControlStateNormal]
                   fontFamily:fontFamily
                     fontSize:fontSize
                   fontWeight:fontWeight
                        color:textColor
-                    centered:NO];
+                    centered:TRUE];
     [self setTitleAttributes:tabItem titleAttributes:normalAttributes];
 
     NSDictionary *selectedAttributes = [RNNFontAttributesCreator
@@ -106,7 +112,7 @@
                     fontSize:fontSize
                   fontWeight:fontWeight
                        color:selectedTextColor
-                    centered:NO];
+                    centered:TRUE];
     [self setSelectedTitleAttributes:tabItem selectedTitleAttributes:selectedAttributes];
 }
```

**Comment by asafkorem** [comment]
Thanks for the patch suggestion @phuongwd, I will check that now 🙂

**Comment by asafkorem** [comment]
Fix is here: https://github.com/wix/react-native-navigation/pull/7863, will release a patch version soon

**Comment by phuongwd** [comment]
> Thanks for the patch suggestion @phuongwd, I will check that now 🙂

Credit to https://github.com/wix/react-native-navigation/issues/7506#issuecomment-1600381275

**Comment by asafkorem** [comment]
The fix will be released in the next patch version. @phuongwd you will both get credit in the release notes 😄

**Comment by asafkorem** [comment]
Released with [7.38.4](https://github.com/wix/react-native-navigation/releases/tag/7.38.4)
